content/guides/nextjs/adding-tina/adding-tina-provider.md

```diff
@@ -34,12 +34,15 @@ function MyApp({ Component, pageProps }) {
   return <Component {...pageProps} />
 }
 
-export default withTina(MyApp)
+export default withTina(MyApp, {
+  enabled: process.env.NODE_ENV !== 'production',
```

Comment:
```suggestion
  enabled: true,
```
I think, for simplicity's sake, let's just set this to `true`. We've made it easier to enable/disable the CMS and have found successful workflows for running Tina in production.